### PR TITLE
Add Nyquist Rev. 2

### PR DESCRIPTION
**Pull request by nooges**

### CODE REVIEW

File: keyboards/nyquist/rev2/rev2.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#include "rev1.h"
```
Comment (jackhumbert) [inline keyboards/nyquist/rev2/rev2.c]: Looks like you just need a `rev2.h` here and this should be ready to go :)

File: keyboards/nyquist/rev2/rev2.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#ifdef SSD1306OLED
+void led_set_kb(uint8_t usb_led) {
+    // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+    led_set_user(usb_led);
+}
+#endif
+
+void matrix_init_kb(void) {
+
+    #ifdef AUDIO_ENABLE
```
Comment (drashna) [inline keyboards/nyquist/rev2/rev2.c]: This section will break startup sounds.  They're called elsewhere, and override the behavior, removing the option for user configuration (eg config.h override)

File: keyboards/nyquist/rev2/rev2.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    // // orange led on
+    // DDRB |= (1<<0);
+    // PORTB &= ~(1<<0);
+
+	matrix_init_user();
+};
+
+void shutdown_user(void) {
+    #ifdef AUDIO_ENABLE
```
Comment (drashna) [inline keyboards/nyquist/rev2/rev2.c]: same as other.  Breaks shutdown chime

**Comment by drashna** [comment]
Looks like this is actually missing the rev2.h include in the `nyquist.h` file, to none of the rev2 code is getting called.

**Comment by nooges** [comment]
@drashna Ready for re-review

**Comment by drashna** [comment]
Awesome, looks good!

**Pull request merged**